### PR TITLE
MEV bot

### PR DESCRIPTION
**Pull request by acenolaza**
* Updates all-postitions.json
* Removes _E18 suffix on MIN_POSITION_USD env var
* profitReceiver to benefeciary in liquidator contract
* Minor improvements and fixes links on README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,22 +10,22 @@ result of the OEV bot, there are branches for the previous steps, so one can eas
 On branch [mev](https://github.com/api3dao/oev-v1-compound-bot/tree/mev)
 
 First step is to have an MEV bot that can perform liquidations when the opportunity to do so arises. For more
-information, refer to [OEV Searching](https://docs.api3.org/oev/searchers/).
+information, refer to [MEV Searching](https://docs.api3.org/oev-searchers/in-depth/#from-mev-searching).
```
Comment (Siegrift) [inline README.md]: I think OEV Searching is a bit more accurate.

**Review by Siegrift — APPROVED** [review]
👍 LGTM

I think the README wording a bit simpler the way you wrote it.

File: README.md (hunk 1 above)
Comment (acenolaza) [reply README.md]: Yes, true. I guess I was trying to differentiate this docs link with the third one (which is also OEV Searching). What do you thing of using "From MEV Searching" instead?

**Comment by acenolaza** [comment]
I see that each branch have a single commit so should I go ahead and merge this PR then update the other branches or is there a specific process that I should follow for pushing to this repo?

**Comment by Siegrift** [comment]
> I see that each branch have a single commit so should I go ahead and merge this PR then update the other branches or is there a specific process that I should follow for pushing to this repo?

We don't necessarily need to maintain the one commit property. The only thing we want is the diffs between branches to show up nicely (so that the diff from migrating from MEV bot to "MEV with signed data" and from "MEV with signed data" to OEV bot shows up nicely.

File: README.md (hunk 1 above)
Comment (acenolaza) [reply README.md]: Used "From MEV Searching" and also added a link to the oev-v1-compound repo.

**Pull request merged**